### PR TITLE
Fixed duplicate boxplots

### PR DESCRIPTION
**Pull request by kwongz**
### Checklist

- [x] For UI or styling changes, I have added a screenshot or gif showing before & after
- [x] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)
- ~~I have added to the docs where applicable~~
- ~~I have added to the [VS Code extension](https://github.com/evidence-dev/evidence-vscode) where applicable~~

https://github.com/user-attachments/assets/76df5439-e852-4e37-a335-2a06371c28f9

### CODE REVIEW

**Comment by ItsMeBrianD** [comment]
Needs changeset + linting error. Is removing `data.fetch` useful here?

**Comment by kwongz** [comment]
@ItsMeBrianD I noticed the boxplot stories console logging the data differently then in the local evidence app.

I wrapped Boxplot stories with a QueryLoad whcih fixed the different behaviour for me. I recall there being a change to handling Query's maybe a month ago? Maybe that's what this bug appeared?

Would like to talk more about this when you have the chance

No Queryload wrapper

https://github.com/user-attachments/assets/79cff482-7b88-490f-a294-c6cd8cf63696

Queryload wrapper
https://github.com/user-attachments/assets/aa0d4447-31b0-4050-b1e5-fe076ed01140

**Comment by kwongz** [comment]
fix #2462

**Comment by kwongz** [comment]
@ItsMeBrianD after taking a second look, the original async function had updateBoxPlotData();, which would make it run independently, cause the data to render duplicates?

Removing the updateBoxplotdata(); from async solves the duplicate issue, but the component also seems to run fine without the data.fetch();

**Comment by zachstence** [comment]
> I think having `QueryLoad` required in the story is a smell here - that behavior should be handled inside the component. Might be related to why `data.fetch` was being used initially.
> 
> Might check with @zachstence to see what this PR was related to [a4c937e](https://github.com/evidence-dev/evidence/commit/a4c937ec9449b5d6841c43b7515482bc22db44fe)

That change was to fix BoxPlot not rendering in storybook. I made that change as part of cleaning things up for Chromatic

**Comment by kwongz** [comment]
Boxplot stories, no queryload


https://github.com/user-attachments/assets/c28d3c42-dad4-4091-93c6-93d79c9ec921

File: packages/ui/core-components/src/lib/unsorted/viz/box/BoxPlot.stories.svelte [hunk 1]
```diff
@@ -157,6 +157,7 @@ limit 50`,
 	}}
 	let:args
 >
+	<!-- {console.log([...flightData])} -->
```
Comment (ItsMeBrianD) [inline packages/ui/core-components/src/lib/unsorted/viz/box/BoxPlot.stories.svelte:160]:
```suggestion
```

**Review by ItsMeBrianD — APPROVED** [review]
Remove console log then good

**Pull request merged**